sway/input/keyboard.c

```diff
@@ -278,9 +278,7 @@ static void handle_keyboard_key(struct wl_listener *listener, void *data) {
 				raw_modifiers, false, input_inhibited);
 
 		if (binding_pressed) {
-			if ((binding_pressed->flags & BINDING_RELOAD) == 0) {
-				next_repeat_binding = binding_pressed;
-			}
+			next_repeat_binding = binding_pressed;
```

Comment:
Since `next_repeat_binding` isn't being set conditionally now, one of these variables can be removed.